source_test.go

```diff
@@ -195,7 +206,7 @@ func testSourceRateLimit(t *testing.T, cfg map[string]string) {
 func openTestSource(t *testing.T, cfg map[string]string) sdk.Source {
 	is := is.New(t)
 
-	s := NewSource()
+	s := &Source{}
```

Comment:
Any special reason for this?

Reply:
Yes, when testing `Read` we can't test that the connector produces structured data if the middleware encodes it into avro 🙈 I mean we could, but we would really be testing the middleware at that point, so now we only test what the connector produces.